labeling/internal/python.go

```diff
@@ -47,6 +48,13 @@ var PythonRules = []labels.Rule{
 		pipfile, _ := c.FindFile(pipenvFiles...)
 		label.Valid = pipfile != ""
 		label.BasePath = path.Dir(pipfile)
+
+		pyprojectPath, _ := c.FindFile("pyproject.toml")
+		if pyprojectPath != "" && fileContainsString(c, pyprojectPath, "pipenv") {
```

Comment:
Did you consider using a toml parser?
We might want one for rust anyways.

Reply:
Since the plan was just to look for the specific string and not a specific key I didn't consider using it. I think we can look into a better solution if we notice that this is still incorrectly choosing the package manager 🙂